.github/workflows/codspeed.yml

```diff
@@ -0,0 +1,48 @@
+name: Benchmarks
+
+on:
+  push:
+    branches:
+      - 'master'
+      - 'next'
+    paths:
+      - 'packages/x-charts*/**'
```

Comment:
We could change this to "release" instead, to update the base "bechmarks" release-by-release instead of every push to master